code/game/gamemodes/gang/gang.dm

```diff
@@ -67,50 +67,23 @@ GLOBAL_LIST_INIT(gang_outfit_pool, list(/obj/item/clothing/suit/jacket/leather,/
 		SSjob.DisableJob(/datum/job/cyborg)
 
 		// For removing problematic items
-		for(var/area/ai_monitored/security/armory/A in GLOB.sortedAreas)
-			target_armory = A
-			break
-
-		for(var/area/crew_quarters/heads/hos/S in GLOB.sortedAreas)
-			target_hos = S
-			break
-
-		for(var/area/security/brig/B in GLOB.sortedAreas)
-			target_brig = B
-			break
-
-		for(var/area/security/main/M in GLOB.sortedAreas)
-			target_equip  = M
-			break
-
-		for(var/area/security/warden/W in GLOB.sortedAreas)
-			target_ward = W
-			break
-
-		for(var/area/security/detectives_office/D in GLOB.sortedAreas)
-			target_det = D
-			break
-
-		for(var/area/crew_quarters/heads/captain/private/V in GLOB.sortedAreas)
-			target_captain = V
-			break
+		target_armory = locate(/area/ai_monitored/security/armory in GLOB.sortedAreas)
```

Comment:
I don't understand how this is compiling but what I meant was: `target_armory = locate(/area/ai_monitored/security/armory) in GLOB.sortedAreas`